game/scripts/vscripts/abilities/fountain_attack.lua

```diff
@@ -58,12 +58,20 @@ modifier_fountain_attack_aura = class(ModifierBaseClass)
 
 function modifier_fountain_attack_aura:OnCreated(keys)
   local caster = self:GetCaster()
+  local teamID caster:GetTeamNumber()
```

Comment:
Missing an `=` there. Should be `local teamID = caster:GetTeamNumber()`. Classic Chrono. :P

Reply:
dota didn't threw any errors

Reply:
I dunno. I just tried putting a line like that in my editor with the linter enabled. Apparently it's not a syntax error, cause Lua. And Dota apparently doesn't have any issues with you calling CreateParticle with a `nil` name. You just don't get a particle.